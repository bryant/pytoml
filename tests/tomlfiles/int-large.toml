largepos = 9223372036854775808   # 2**63
largeneg = -9223372036854775808  # -2**63
