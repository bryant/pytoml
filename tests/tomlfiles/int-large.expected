{'largepos': 9223372036854775808, 'largeneg': -9223372036854775808}
